prioritize.py

```diff
@@ -26,12 +30,95 @@
 )
 
 
+# Priorities
+prioritiesT = List[float]
```

Comment:
`Priorities` And `PrioritiesNormalized`  according to https://peps.python.org/pep-0484/